docs/using-connector.md

```diff
@@ -77,6 +78,17 @@ conf.set("spark.riak.connections.max", "50")
 sc = pyspark.SparkContext("spark://127.0.0.1:7077", "test", conf)
 ```
 
+
+## Failover Handling
+ 
+When reading data from Riak, the Spark-Riak connector will run a full bucket query. This query first obtains metadata, called a coverage plan, from Riak about 
```

Comment:
Instead of referring to Riak KV (the product), let's say "When reading data from a Riak KV bucket..."